changelog.d/20240816_093119_athornton_DM_45805.md

```diff
@@ -0,0 +1,5 @@
+<!-- Delete the sections that don't apply -->
+
+### Other changes
```

Comment:
Since we changed the default, I think this is a backwards-incompatible change (it will require attention from people running Nublado if they don't want `/tmp` to be a memory file system). The next release would therefore be 7.0.0.